exercises/grep/description.md

```diff
@@ -1,77 +1,25 @@
-# Description
+# Instructions
 
-Search a file for lines matching a regular expression pattern. Return the line
-number and contents of each matching line.
+Search files for lines matching a search string and return all matching lines.
 
-The Unix [`grep`](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/grep.html) command can be used to search for lines in one or more files
-that match a user-provided search query (known as the *pattern*).
+The Unix [`grep`](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/grep.html) command searches for lines in files that match a regular expression. Your task is to implement a simplified `grep` function, which supports searching for literal strings.
 
-The `grep` command takes three arguments:
+The `grep` function takes three arguments:
 
-1. The pattern used to match lines in a file.
-2. Zero or more flags to customize the matching behavior.
-3. One or more files in which to search for matching lines.
+1. The string to search for.
+2. Zero or more flags to customize its behavior.
```

Comment:
In English this `its` refers to `flags`, not `grep`. Repeating "the matching" resolves that. But if you say `matching`, then `pattern` is perhaps a better candidate because of pattern matching.

Reply:
Is it not true that since "flags" is plural it would need to be referred to with "their", and thus cannot be the target of "its"? Anyway, since it is potentially confusing, I've replaced "its" with "the command's".

Reply:
Eh. Yeah you're right. I'm that case it would refer to the last singular noun which is not the command so your solution solves all of it. 🙏🏽